src/faldbt/project.py

```diff
@@ -172,18 +173,21 @@ def __hash__(self) -> int:
     def get_depends_on_nodes(self) -> List[str]:
         return self.node.depends_on_nodes
 
-    def get_post_hook_paths(self, keyword: str = "fal") -> List[str]:
+    def _get_hook_paths(self, keyword: str) -> List[str]:
```

Comment:
We need the `keyword="fal"` param because we have the `--keyword` flag. Can you re-add it and add a new param for the `hook_keyword`?

Reply:
Good catch! I search through the code base for `get_post_hook_paths` and none of the uses was passing `keyword` so I thought it was obsolete but I think it is a bug right?

https://github.com/fal-ai/fal/blob/d2a2022a292989a83d9ea767b8b3f9e00a041605/src/fal/cli/fal_runner.py#L88

E.g. should the use case above pass `faldbt.keyword`?

Reply:
probably!